docs/pages/User/Install/_index.md

```diff
@@ -1,24 +1,134 @@
+echo "POD CIDR: $POD_CIDR"
+echo "SERVICE CIDR: $SERVICE_CIDR"
+```
+{{% /expand%}}
+
+{{%expand "AWS Elastic Kubernetes Service (EKS)" %}}
+Retrieve information about your clusters, typing:
+```bash
+API_SERVER=$(aws eks describe-cluster --name liqo-cluster --region=eu-central-1 | jq -r .cluster.endpoint | sed 's/https:\/\///g' ))
+
+```
+{{% /expand%}}
+
+{{%expand "Azure Kubernetes Service (AKS)" %}}
+
+AKS clusters have by default with the following PodCIDR and ServiceCIDR:
+
+If you are using Azure CNI:
+
+```bash
+SUBNET_ID=$(az aks list --query="[?name=='__YOUR_CLUSTER_NAME__']" | jq -r '.[0].agentPoolProfiles[0].vnetSubnetId')
+POD_CIDR=$(az network vnet subnet show --ids ${SUBNET_ID} | jq -r .addressPrefix)
+SERVICE_CIDR=$(az aks list --query="[?name=='__YOUR_CLUSTER_NAME__']" | jq -r ".[0].networkProfile.serviceCidr")
+```
+
+Or Kubenet:
+
+```bash
+POD_CIDR=$(az network vnet subnet show --ids ${SUBNET_ID} | jq -r ".[0].networkProfile.serviceCidr)
+SERVICE_CIDR=$(az aks list --query="[?name=='__YOUR_CLUSTER_NAME__']" | jq -r ".[0].networkProfile.serviceCidr"
+```
+
+
+{{% /expand%}}
+{{%expand "Google Kubernetes Engine (GKE)" %}}
+
+```bash
+SERVICE_CIDR=$(gcloud container clusters describe __YOUR_CLUSTER_NAME__ --zone -__YOUR_ZONE__ --project __YOUR_PROJECT_ID__ --format="json" | jq -r `.servicesIpv4Cidr`)
+POD_CIDR=$(gcloud container clusters describe __YOUR_CLUSTER_NAME__ --zone -__YOUR_ZONE__ --project __YOUR_PROJECT_ID__ --format="json" | jq -r `.clusterIpv4Cidr`)
+API_SERVER=$(gcloud container clusters describe __YOUR_CLUSTER_NAME__ --zone -__YOUR_ZONE__ --project __YOUR_PROJECT_ID__ --format="json" | jq -r `.endpoint`)
+```
+
+{{% /expand%}}
+{{%expand "K3s" %}}
+
+K3s clusters have by default with the following PodCIDR and ServiceCIDR:
+
+| Variable               | Default | Description                                 |
+| ---------------------- | ------- | ------------------------------------------- |
+| `networkManager.config.podCIDR`             |    10.42.0.0/16     |
+| `networkManager.config.serviceCIDR`         |    10.43.0.0/16     |
+{{% /expand%}}
+
+#### Set-Up Liqo Repository
+
+Firstly, you should add the official Liqo repository to your Helm Configuration:
+
+```bash
+helm repo add liqo https://helm.liqo.io/
+```
+
+#### Set-up
+
+The most important values you can set are the following:
+
+| Variable               | Description                                 |
+| ---------------------- | ------------------------------------------- |
+| `networkManager.config.podCIDR`        | The cluster Pod CIDR                                 |
+| `networkManager.config.serviceCIDR`    | The cluster Service CIDR                             |
+| `discovery.config.clusterLabels`       | Labels used to characterize your cluster's resources |
+| `auth.config.allowEmptyToken`          | Enable/disable [cluster pre-authentication](/User/Configure/Authentication)            |
```

Comment:
Better name? E.g., "auth.config.allowNullAuth"? Or "auth.config.disableAuth"?

Reply:
`auth.config.allowNullAuth` sounds good for me